index.html

```diff
@@ -5,105 +5,125 @@
         <meta http-equiv="X-UA-Compatible" content="chrome=1">
         <meta name="viewport" content="width=640" />
 
-        <link rel="stylesheet" href="web/stylesheets/core.css" media="screen"/>
-        <link rel="stylesheet" href="web/stylesheets/mobile.css" media="handheld, only screen and (max-device-width:640px)"/>
-        <link rel="stylesheet" href="web/stylesheets/pygment_trac.css"/>
+        <link rel="stylesheet" href="web/stylesheets/main.css"/>
+        <link href="https://fonts.googleapis.com/css?family=Open+Sans:300&display=swap" rel="stylesheet">
```

Comment:
Can you please download that and move that to `web/fonts/` please. So we don't expose the visitors to Google tracking.